link/src/main/java/com/stripe/android/link/ui/wallet/ConfirmRemoveDialog.kt

```diff
@@ -0,0 +1,54 @@
+package com.stripe.android.link.ui.wallet
+
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.stringResource
+import com.stripe.android.link.R
+import com.stripe.android.link.model.removeConfirmation
+import com.stripe.android.link.theme.linkColors
+import com.stripe.android.model.ConsumerPaymentDetails
+
+@Composable
+internal fun ConfirmRemoveDialog(
```

Comment:
This was previously located in the deleted `WalletModals.kt` file. Nothing was changed except the file name.